src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx

```diff
@@ -196,23 +220,36 @@ export class PluggableGeoPushpinChart extends PluggableBaseChart {
         };
     }
 
-    private getMeasureSizeBucketItem(buckets: IBucket[]): IBucketItem[] {
-        if (hasBucket(buckets, BucketNames.MEASURES)) {
-            return getBucketItemsByType(buckets, BucketNames.MEASURES, [METRIC]).slice(0, 1);
+    private getSegmentItems(buckets: IBucket[]): IBucketItem[] {
+        let segments = getPreferredBucketItems(
+            buckets,
+            [BucketNames.STACK, BucketNames.SEGMENT, BucketNames.COLUMNS],
+            [ATTRIBUTE],
+        );
+        const nonSegmentAttributes = getAttributeItemsWithoutStacks(buckets);
+        if (nonSegmentAttributes.length > 1 && isEmpty(segments)) {
+            // first attribute is taken, find next available non-date attribute
+            const attributesWithoutFirst: IBucketItem[] = tail(nonSegmentAttributes);
```

Comment:
it could be 
```
const [_firstItem, ...attributes] = nonSegmentAttributes;
```

Reply:
I sugeest :
`const [, ...attributes] = nonSegmentAttributes;`

Reply:
There is a warning about `_firstItem` is declared but never used.
So i'd like to keep the current one.